EXPLAINER_BLUETOOTH.md

```diff
@@ -220,6 +220,25 @@ applications identify whether a connection attempt is likely to succeed, a
 `SerialPort.connected` attribute will be added to indicate whether the port is
 logically connected.
 
+The below examples shows how `SerialPort.connected` attribute can be used to
+inform user right actions if the Bluetooth device is not powered on or not in
+the proximity of the system.
+
+```javascript
+// Assuming we have `port`, which is a wireless serial port.
+try {
+   await port.open({baudRate: 115200});
+} catch(e) {
+   if (!port.connected) {
```

Comment:
Sorry to ask you to rewrite this, but I think a use case which better highlights where this new attribute is useful is if the site is using `getPorts()` to check what devices are available and potentially connect to them automatically.

Without this attribute you have to call `port.open()` blindly, like in your example. With the `connected` attribute and event you can call `port.open()` only when `connected` is true, and otherwise show the port to the user and provide the option to connect (which might fail).

This matches the use case for the developer we worked with when designing this change.

Reply:
I see. Thanks for the suggestion. How about an example like this?

```javascript
let ports = await navigator.serial.getPorts();
for (let port of ports) {
  if (port.connected) {
    await port.open({baudRate: 115200});
  } else {
    // Prompt the user to make sure the Bluetooth device is connectable to the system.
    // Once the user thinks the device is ready to connect, the user can press a
    // a button which runs `port.open` to trigger a connection attempt.
}
```

Reply:
Yeah, that looks good. Thanks!

Reply:
Updated the pull request.